### DISCUSSION TITLE
Consistent face for different TODO keywords

### DISCUSSION THREAD

**Discussion by kaibagley** [minad/org-modern#244] [Q&A] [answered]
Hi,

I'm not much of an elisp wiz so sorry if I'm missing something obvious here

I recently updated (can't remember how long my last update was, 1-2 months?) and it made all the keywords look inconsistent (TODO had the semibold default todo face, and my custom keywords looked the same as usual).

Now I'm trying to change the custom keywords to the new semibold style with little success:

```elisp
;; Org modern
(use-package org-modern
  :config
  (setq org-modern-block-fringe nil
        org-modern-list '((43 . "◦") (45 . "•") (42 . "•"))
        org-modern-priority-faces '((?A :background "#291f26" :foreground "#b52c2c")
                                    (?B :background "#291f26" :foreground "#b08940")
                                    (?C :background "#291f26" :foreground "#bf9f8f"))
        ;; org-modern-tag '((t :inherit org-modern-label :background "#291f26" :foreground "#bf9f8f"))
        org-modern-todo-faces '(("NEXT" :inherit org-modern-todo :foreground "dodger blue")
                                ("WAIT" :inherit org-modern-todo :foreground "orange")
                                ("INTR" :inherit org-modern-todo :foreground "#fdb4eb"))
        )
  (if (display-graphic-p)
      (setq org-modern-table t)
    (setq org-modern-table nil))
  (global-org-modern-mode))
```

Using this config I get this:

![Capture](https://github.com/user-attachments/assets/b4cc1091-81e0-489c-82cc-9c71647497fd)

Which is close, but the font size is different, why is this happening even though it's inherited from the TODO face?

Also while you're here, how can I get the commented `org-modern-tag` line to work, I simply want my tags to all have that fore- and background

Thanks a lot!!

**Comment by minad** [comment]
Hi. Can you try out different themes to see if the problem occurs there too? The Ef and Modus themes usually work well.

**Reply by kaibagley** [reply]
Here it is with theme "ef-rosa". My original screenshot was "ef-cherie"

![image](https://github.com/user-attachments/assets/0b9ec40f-e46f-4c80-aed2-74426e47adb2)

**Reply by kaibagley** [reply]
It also occurs in a fresh emacs (`emacs -Q`)

![image](https://github.com/user-attachments/assets/8604e84b-0e91-496f-96d8-baf7ae54071c)

**Comment by kaibagley** [comment, marked as answer]
I can achieve it using the following:

```
;; Org modern
(setq org-modern-todo-faces '(("NEXT" :inverse-video t :weight semibold :foreground "dodger blue")
                              ("WAIT" :inverse-video t :weight semibold :foreground "orange")
                              ("INTR" :inverse-video t :weight semibold :foreground "#fdb4eb")))

```
